### PR TITLE
Nd tabbed interface

### PR DESCRIPTION
**Pull request by SaraDawner2000**
Complete part 4.2 of industrial photogram

### CODE REVIEW

**Review by rayguna — APPROVED** [review]

File: app/views/users/following.html.erb
Comment (rayguna) [inline app/views/users/following.html.erb:1]: A neat use of leaders encapsulated method within the User class.

File: app/views/users/feed.html.erb
Comment (rayguna) [inline app/views/users/feed.html.erb:1]: A neat use of feed encapsulated method within the User class.

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,5 +1,22 @@
 class UsersController < ApplicationController
+  before_action :set_user
```
Comment (CarlosCastanedaDev) [inline app/controllers/users_controller.rb:2]: Great job using the before_action and keeping your code DRY!

**Review by CarlosCastanedaDev — APPROVED** [review]
Great job, Natalie! Good use of a partial, you can now merge theses changes back to main,

**Pull request merged**